library/core/src/array/mod.rs

```diff
@@ -31,13 +31,11 @@ pub use iter::IntoIter;
 /// # Example
 ///
 /// ```rust
-/// #![feature(array_from_fn)]
-///
 /// let array = core::array::from_fn(|i| i);
 /// assert_eq!(array, [0, 1, 2, 3, 4]);
 /// ```
 #[inline]
-#[unstable(feature = "array_from_fn", issue = "89379")]
+#[stable(feature = "array_from_fn", since = "1.61.0")]
 pub fn from_fn<F, T, const N: usize>(mut cb: F) -> [T; N]
```

Comment:
i think we should change the generic arguments to
```suggestion
pub fn from_fn<T, const N: usize, F>(mut cb: F) -> [T; N]
```
this makes it a lot more similar to other such functions with the function being last.

Reply:
We might even consider going to

```suggestion
pub fn from_fn<T, const N: usize>(mut cb: impl FnMut(usize) -> T) -> [T; N]
```

Because if the rules change to "you can turbofish the explicit generic parameters but not the APITs", then that would be nicer -- `from_fn::<i32, N>(...)` instead of `::<i32, N, _>`.

That would, of course, mean that they'd need to be provided by context in the mean time, however.  But TBH I prefer that anyway, as I find `from_fn::<_, 4, _>` distasteful.  (The closure almost always provides the `T`.)